_sass/reuse.scss

```diff
@@ -0,0 +1,18 @@
+#services {
+  h2 {
+      max-width: 60rem;
```

Comment:
Probably `em` rather than `rem`: I guess you want the title to be limited to about 60 characters of its size, not to a varying amount of characters depending on the font size, right?